.github/workflows/push-training-ecs.yml

```diff
@@ -78,20 +78,24 @@ jobs:
           # Build a docker container and
           # push it to ECR so that it can
           # be deployed to ECS.
-          docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG training -f training/Dockerfile.prod
+          docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG training -f training/training/core/celery/Dockerfile
```

Comment:
> training/training
Just to clarify, this path is not a typo?

Reply:
yes, since it runs from the root of the repo

Reply:
gotcha